examples/linear/index.js

```diff
@@ -0,0 +1,100 @@
+const express = require('express');
+const everyauth = require('@fusebit/everyauth-express');
+const { LinearClient } = require('@linear/sdk');
+const { v4: uuidv4 } = require('uuid');
+const cookieSession = require('cookie-session');
+
+const app = express();
+app.use(express.json());
+app.use(express.urlencoded({ extended: true }));
+const port = 3000;
+
+// Get userId from the authorization redirect or via session if already authorized.
+const handleSession = (req, res, next) => {
+  if (req.query.userId) {
+    req.session.userId = req.query.userId;
+  }
+  if (!req.session.userId) {
+    return res.redirect('/');
+  }
+  return next();
+};
+
+app.use(
+  cookieSession({
+    name: 'session',
+    secret: 'secret',
+  })
+);
+
+app.set('view engine', 'pug');
+
+app.get('/', (req, res) => {
+  if (!req.session.userId) {
+    return res.redirect(`/authorize/${uuidv4()}`);
+  }
+  res.redirect('/finished');
+});
+
+app.use(
+  '/authorize/:userId',
+  (req, res, next) => {
+    if (!req.params.userId) {
+      return res.redirect('/');
+    }
+    return next();
+  },
+  everyauth.authorize('linear', {
+    // The endpoint of your app where control will be returned afterwards
+    finishedUrl: '/finished',
+    // The user ID of the authenticated user the credentials will be associated with
+    mapToUserId: (req) => req.params.userId,
+  })
+);
+
+app.post('/create-issue', handleSession, async (req, res) => {
+  const userCredentials = await everyauth.getIdentity('linear', req.session.userId);
```

Comment:
Might be a good idea to add a comment saying something along the lines of using this will ensure you always have the latest freshest access token for any call to make it Linear.